sonic_installer/bootloader/grub.py

```diff
@@ -164,7 +164,8 @@ def is_secure_upgrade_image_verification_supported(self):
             if ! [ -n "$(ls -A /sys/firmware/efi/efivars 2>/dev/null)" ]; then
                 mount -t efivarfs none /sys/firmware/efi/efivars 2>/dev/null
             fi
-            SECURE_UPGRADE_ENABLED=$(bootctl status 2>/dev/null | grep -c "Secure Boot: enabled")
+            SECURE_UPGRADE_ENABLED=$(mokutil --sb-state 2>/dev/null | \
+                grep -c "enabled") || CHECK_SECURE_UPGRADE_ENABLED=$?
```

Comment:
From where are you getting the "CHECK_SECURE_UPGRADE_ENABLED" variable?

Reply:
@Yarden-Z  Remove as unneeded